### PR TITLE
User profile improvements

### PR DESCRIPTION
**Pull request by np**

### CODE REVIEW

**Comment by andorp** [comment]
LGTM

**Comment by np** [comment]
Thanks for the review of the two first commints but this is not finished yet, I've assigned myself.

**Comment by andorp** [comment]
Relates to: https://github.com/liqd/aula/issues/129

File: src/Frontend/Core.hs [hunk 1]
```diff
@@ -180,8 +180,7 @@ makeLenses ''Frame
 -- for more details).
 type GetH = Get '[HTML, PlainText]
 type PostH = Post '[HTML] ()
-type FormHandlerT p a = FormH '[HTML, PlainText] (Frame (FormPageRep p)) a
```
Comment (andorp) [inline src/Frontend/Core.hs]: \o/

**Comment by fisx** [comment]
LGTM.  lots of cleanup work, thanks!  I have two nit-pick comments and will merge when green.

**Comment by andorp** [comment]
LGTM

**Pull request merged**